app/src/main/java/com/example/poste/api/poste/models/Folder.java

```diff
@@ -119,4 +128,8 @@ public boolean update() {
             return API.updateFolder(this.id, this.name, this.ownerId);
         } catch (Exception e) { return false; }
     }
+
+    public void addPost(Post post) {
+        posts.add(post);
```

Comment:
See comments above re: API call for this functionality to avoid client-side / backend sync issues

Reply:
Discussed below.